draft-packages/modal/KaizenDraft/Modal/Primitives/ModalFooter.scss

```diff
@@ -1,29 +1,41 @@
 @import "~@kaizen/deprecated-component-library-helpers/styles/layout";
 @import "~@kaizen/deprecated-component-library-helpers/styles/type";
 @import "~@kaizen/deprecated-component-library-helpers/styles/color";
+@import "~@kaizen/component-library/styles/responsive";
 
 .actions {
   display: flex;
   flex-direction: row-reverse;
-  justify-content: center;
+  justify-content: flex-start;
 }
 
 .actionsAlignStart {
   flex-direction: row-reverse;
   justify-content: flex-end;
 }
 
-.informationAlign {
-  flex-direction: row;
-  justify-content: flex-start;
-
-  .actionButton + .actionButton {
-    @include ca-margin($start: $ca-grid * 0.5);
+.padded {
+  padding: 0 $ca-grid * 1.5 $ca-grid * 1.5 $ca-grid * 1.5;
+  @include ca-media-mobile {
+    padding: $ca-grid;
+    flex-direction: column;
+  }
+}
+.actionButton {
+  @include ca-media-mobile {
+    width: 100%;
+    > *,
+    > * > * {
```

Comment:
I might be missing something here, but do you need _both_ of these selectors?

Reply:
Great question - I will double check what my brain was trying to achieve here

Reply:
Removed this styling for the button width and used `useMediaQueries` to enable the fullwidth prop for the button component